packages/ui-popover/src/Popover/README.md

```diff
@@ -170,6 +226,148 @@ class Example extends React.Component {
 render(<Example />)
 ```
 
+#### Popover playground
+
+```js
+---
+render: false
+example: true
+---
+class Example extends React.Component {
+
+  state = {
+    shouldAlignArrow: true,
+    isShowingContent: true,
+    withArrow: true,
+    placement: 'top',
+    shadow: 'topmost',
+    placementsValues: [
+      'top',
+      'end',
+      'bottom',
+      'start',
+      'top start',
+      'start top',
+      'start center',
+      'start bottom',
+      'bottom start',
+      'bottom center',
+      'bottom end',
+      'end bottom',
+      'end center',
+      'end top',
+      'top end',
+      'top center',
+      'center end',
+      'center start'
+    ],
+    shadowValues: [
+      'none',
+      'resting',
+      'above',
+      'topmost'
+    ],
```

Comment:
As Topi said, these shouldn't be in the state, because we never change them. You could move them to the class, and then you can use them as `this.shadowValues`. 

```js
state = { ... }
shadowValues = [ ...  ]

// and then later in the render:
{
  this.shadowValues.map((shadow, index) => ( ... ))
}
```

Reply:
I have change them to static variables